mockito2/src/test/java/com/baeldung/mockito/java8/LazyVerificationTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.mockito.java8;
```

Comment:
mockito2 module is now in "testing-modules", let's move it there

Reply:
I moved it testing-modules as suggested.